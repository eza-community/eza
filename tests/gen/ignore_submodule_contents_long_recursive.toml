bin.name = "eza"
args = "tests/test_dir/git/with_submodule -R --long --ignore-submodule-contents"
