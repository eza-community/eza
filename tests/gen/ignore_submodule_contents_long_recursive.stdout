drwxr-xr-x - nixbld  1 Jan  1970 001
.rw-r--r-- 2 nixbld  1 Jan  1970 file_000
.rw-r--r-- 2 nixbld  1 Jan  1970 file_001
.rw-r--r-- 2 nixbld  1 Jan  1970 file_002
.rw-r--r-- 2 nixbld  1 Jan  1970 file_003
.rw-r--r-- 2 nixbld  1 Jan  1970 file_004
.rw-r--r-- 2 nixbld  1 Jan  1970 file_005
.rw-r--r-- 2 nixbld  1 Jan  1970 file_006
.rw-r--r-- 2 nixbld  1 Jan  1970 file_007
.rw-r--r-- 2 nixbld  1 Jan  1970 file_008
.rw-r--r-- 3 nixbld  1 Jan  1970 file_009

tests/test_dir/git/with_submodule/001:
